catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraCreators.scala

```diff
@@ -49,16 +47,20 @@ trait SierraCreators extends MarcUtils {
         val name = subfields.collectFirst {
           case MarcSubfield("a", content) => content
         }
-        val codes = subfields.collect {
-          case MarcSubfield("0", content) => content
-        }
+        val codes = getIdentifierCodes(subfields)
         val organisation = Organisation(name.get)
         identify(codes, organisation, "Organisation")
       }
 
     persons ++ organisations
   }
 
+  private def getIdentifierCodes(subfields: List[MarcSubfield]) = {
+    subfields.collect {
+      case MarcSubfield("0", content) => content.replaceAll("\\s", "")
```

Comment:
Can we get a comment and example explaining why we're stripping spaces? The one from the PR description would be enough.

Reply:
👍